polyfills/HTMLCollection/prototype/@@iterator/tests.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-env mocha, browser */
+/* global proclaim, Symbol */
+
+function getHTMLCollection() {
+	var fragment = document.createDocumentFragment();
+	fragment.appendChild(document.createElement('div'));
+	fragment.appendChild(document.createElement('div'));
+	return fragment.children;
+}
```

Comment:
```suggestion
var collectionId = 0;
function getHTMLCollection() {
	collectionId++;

	document.body.appendChild(document.createElement('something' + collectionId));
	document.body.appendChild(document.createElement('something' + collectionId));
	return document.getElementsByTagName('something' + collectionId);
}
```

`DocumentFragment.prototype.children` was only added in Edge 16 : https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment/children#browser_compatibility

This makes it less suited in the tests here.

Reply:
Oh cool, I haven't noticed your suggestion while fixing tests. What do you think about my approach?